bin/govuk_taxonomy_supervised_learning_jenkins_script

```diff
@@ -6,11 +6,14 @@ rm -rf ./venv
 virtualenv --python=python3 --no-site-packages ./venv
 export VIRTUAL_ENV="$PWD/venv"
 export PATH="$PWD/venv/bin:$PATH"
+export PYTHONPATH="python"
+
 pip3 install pip==9.0.3
 
 make pip_install
 
-make clean
-make data/taxons.json.gz
-make data/content.json.gz
-make measure_average_taxons
+#make clean
+#make data/taxons.json.gz
+#make data/content.json.gz
+#make measure_average_taxons
```

Comment:
Just a reminder to remove this prior to merging :)

Reply:
👍